README.md

```diff
@@ -102,7 +102,17 @@ nginx is configured to use default port 80.
 
 ### Testing
 
-TBD
+Install pytest for our premade tests.
+
+```bash
+pipenv install pytest
```

Comment:
Nit: `pytest` is probably ok to be installed as a dev-dependency:
```suggestion
pipenv install pytest --dev
```

Reply:
Since pytest is in the Pipfile, won't it get installed already assuming a user followed the beginning of the README to install the project dependencies?

Reply:
I would add a comment about installing it if only because it is required to run it. Mention if previous pipfile installation happened then ignore installation otherwise install it.